src/api/mod.rs

```diff
@@ -355,6 +355,9 @@ pub trait Central: Send + Sync + Clone {
     /// Add a [`Peripheral`] from a MAC address without a scan result. Not supported on all Bluetooth systems.
     async fn add_peripheral(&self, address: &PeripheralId) -> Result<Self::Peripheral>;
 
+    /// Clear the list of [`Peripheral`]s that have been discovered so far.
+    async fn clear_peripherals(&self) -> Result<()>;
```

Comment:
As far as I can see there's no need for this to be async.

Reply:
I think it is best to keep it async, so that if a new platform (or even a linux solution later) needs it to be async we don't need to change the API. What do you think?